pkg/bundle/builder.go

```diff
@@ -239,37 +239,18 @@ func (b *Builder) buildPKIFiles(cluster *kops.Cluster, ig *kops.InstanceGroup, k
 	keys = append(keys, "kube-proxy")
 
 	if ig.IsMaster() {
```

Comment:
The fact that kubelet-api was not in here to be removed suggests that baremetal is not being used.

Reply:
I think @olemarkus was thinking of removing baremetal.

Reply:
Filed #9360